sdk/src/wallet/account/operations/syncing/options.rs

```diff
@@ -43,14 +43,14 @@ pub struct SyncOptions {
     /// Specifies what outputs should be synced for the ed25519 addresses from the account.
     #[serde(default)]
     pub account: AccountSyncOptions,
-    /// Specifies what outputs should be synced for the address of an alias output.
+    /// Specifies what outputs should be synced for the address of an account output.
     #[serde(default)]
     pub alias: AliasSyncOptions,
```

Comment:
Not updated

Reply:
Yeah I'm aware of this one. Not too sure what to do as AccountSyncOptions already exists

Reply:
Maybe AccountOutputSyncOptions or for the other WalletAccountSyncOptions
(or just put a TODO for now, if we change the wallet accounts to only sync outputs for one output and not recursive, then the sync options might also change so we don't have this anymore)

Reply:
I was indeed thinking this would change a bit in the future so I wasn't too bothered about it. I will put a TODO.

Reply:
Done